less/modals.less

```diff
@@ -0,0 +1,114 @@
+//
+// Modals
+// --------------------------------------------------
+
+
+// Shell div to position the modal with bottom padding
+.modal-dialog {
+  position: relative;
+  width: auto;
+  margin: 10px;
```

Comment:
It looks like a lot of this file was copied from less/bootstrap/less/modals.less but doesn't have any changes, e.g. compare this block with https://github.com/chhavijustme/design-system/blob/b3bb71af183221d956c31bde91346ba1277d498e/less/bootstrap/less/modals.less#L43-L48

The original bootstrap .less linked above should already get included via less/styles.less, so here you only need to include the rules that are being changed.

Reply:
Hi, removed the unchanged variables. Hope it looked fine now :) How are we formatting the less files?